crates/core/src/client/config.rs

```diff
@@ -205,5 +207,11 @@ pub async fn get_starknet_account_from_env<P: Provider + Send + Sync + 'static>(
     let chain_id = provider.chain_id().await?;
 
     let local_wallet = LocalWallet::from_signing_key(SigningKey::from_secret_scalar(starknet_account_private_key));
-    Ok(SingleOwnerAccount::new(provider.clone(), local_wallet, starknet_account_address, chain_id))
+    Ok(SingleOwnerAccount::new(
+        provider.clone(),
+        local_wallet,
+        starknet_account_address,
+        chain_id,
+        ExecutionEncoding::Legacy,
```

Comment:
Should we add a comment here to use `New` variant once we migrate to Cairo 1/2 contracts?